.github/utils/update_devcontainer.py

```diff
@@ -0,0 +1,67 @@
+import os
+import argparse
+import json5
+import re
+import logging
+
+logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
+log = logging.getLogger(__name__)
+ECR_REPO = "337532070941.dkr.ecr.us-east-1.amazonaws.com"
+DOCKER_PREFIX = "ugbio"
+IMAGE_REGEX = r'^337532070941\.dkr\.ecr\.us-east-1\.amazonaws\.com/ugbio_[^:]+:[^"]+$'
+
+def is_ugbio_image_used_in_file(file_path):
+    with open(file_path, 'r') as f:
+        data = json5.load(f)
+
+    # Check if the JSON has the "image" field
+    if "image" in data:
+        # Search for the image tag in the file
+        image_match = re.match(IMAGE_REGEX, data["image"])
+        return image_match
+    return False
+        
+def update_image_in_file(file_path, workspace, new_version):
+    new_image = f"{ECR_REPO}/{DOCKER_PREFIX}_{workspace}:{new_version}"
```

Comment:
```suggestion
    new_image = f"{ECR_REPO}/{DOCKER_PREFIX}_{member}:{new_version}"
```

Maybe member instead? to be consistent